### PR TITLE
docs: update docs / shrink images [sc-5276]

### PR DESCRIPTION
**Pull request by tnolet**
- updates docs with newest screens (on our side)
- explains VES, blocking, skipping etc.

### CODE REVIEW

File: site/content/docs/CICD/vercel.md [hunk 1]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
-If you have the `Block my deployment when a check fails` checkbox enabled, Checkly informs Vercel whether a Vercel deployment should be "blocked" based on the following rules, per check type:
+You can block your Vercel deployments if any of your checks fail by enabling the checkboxes highlighted in the
+screenshot below. 
 
-### API checks
-For **API checks** any failing check results in a blocked Vercel deployment. This is a simple, binary state: it either fails or doesn't.
+![blocking deployments](/docs/images/cicd/vercel/vercel_block.png)
 
-### Browser checks
+We have two flavours here...
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:133]: Giga nit: "flavours", "what it says on the tin" -> these colloquialisms might be immediately understandable for someone who speaks English at a comfortable level, but confusing for other folks.

File: site/content/docs/CICD/vercel.md [hunk 2]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
+
+1. **Block my deployment when a check fails**  does what it says on the tin and only apply to Vercel **Reliability checks**.
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:135]: typo? "apply" -> "applies"

File: site/content/docs/CICD/vercel.md [hunk 3]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
+2. **Block my deployment when Web Vitals degrade** is a bit special and only applies to Vercel **Performance checks** for our Browser checks. 
 
-For **Browser checks**, there are different ways a check can fail a Vercel deployment because the Reliability part and Performance part are evaluated separately.
+This table below show help you determine how blocking checks works. Note, again, that **Browser checks**, can fail a
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:138]: typo: "show help you"

Comment (ragog) [reply site/content/docs/CICD/vercel.md:138]: typo: "Note, again, that browser checks, can fail a..."
There is an extra comma in this sentence.

File: site/content/docs/CICD/vercel.md [hunk 4]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
+2. **Block my deployment when Web Vitals degrade** is a bit special and only applies to Vercel **Performance checks** for our Browser checks. 
 
-For **Browser checks**, there are different ways a check can fail a Vercel deployment because the Reliability part and Performance part are evaluated separately.
+This table below show help you determine how blocking checks works. Note, again, that **Browser checks**, can fail a 
+Vercel deployment in two ways because the Reliability part and Performance part are evaluated separately.
 
-**Browser Reliability check**
+| Checkly       | Vercel            | Blocking heuristic                                                                                                                                   |
+|---------------|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| API Check     | Reliability Check | Blocks deployment when it fails due to > 399 HTTP response codes, assertions and other API check specific features                                   |
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:143]: Missing period at the end of sentence.

File: site/content/docs/CICD/vercel.md [hunk 5]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
+| Browser Check | Reliability Check | Blocks deployment when it fails due to > 399 HTTP response codes on the main HTML document, syntax errors in the script, or assertions using the `expect` or `assert` libraries in the browser check script. |
+|               | Performance Check | Blocks deployment when a degradation is registered in the aggregate **Virtual Experience Score** based on Web Vitals. Read below for more details                                   |
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:145]: Missing period at the end of sentence.

File: site/content/docs/CICD/vercel.md [hunk 6]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
-|-----|------------|--------------------|----------|
-| FCP | <= .93 sec | > .93 sec <= 1.6 sec | > 1.6 sec  |
-| LCP | <= 1.2 sec | > 1.2 sec <= 2.4 sec | > 2.4 sec  |
-| CLS | <= 0.1     | > 0.1  <= 0.25     | > 0.25   |
-| TBT | <= 150 ms  | > 150 ms <= 350 ms | > 350 ms |
+1. It aggregates your Web Vitals metrics into one number from 0 to 100, just like you might have seen with [Lighthouse scoring](https://web.dev/performance-scoring/).
+2. It assigns a "weight" to each score, marking its impact on your UX.
+3. It chops up the score into the top 10% (p10), the top 50% (median) and the rest.
 
+In the table below you can see the exact numbers their weight in the Virtual Experience Score.
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:159]: typo: "the exact numbers their weight"

File: site/content/docs/CICD/vercel.md [hunk 7]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
-2. The Web Vitals measurements are within the target range.
 
-We will report it as **blocking** when:
+So, what does this mean for the blocking heuristics? It means that we will mark a check as **blocking** when:
 
 1. A check reports web vitals, and...
-2. The web vitals have degraded in relation to the last available **Production** deploy. Degraded means they passed the range threshold, i.e from **poor** to **needs improvement**.
+2. The VES has degraded in relation to the last available **Production** deploy. Degraded means they passed the range threshold, i.e from **p10** to **median**. 
+
+For more info on the **Virtual Experience Score** [check the documentation on the Vercel site](https://vercel.com/docs/concepts/analytics/web-vitals#virtual-experience-score)
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:174]: Period at end of sentence? I feel you like to skip these after links as they don't look as good somehow.

File: site/content/docs/CICD/vercel.md [hunk 8]
```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
...
+
+## Skipping Performance checks
 
-We will report it as **skipped** when:
+In some cases, Checkly will completely skip performance checks. You will see the "skipped" status in your Vercel deployment overview.
+Checkly skips performance checks when...
 
 1. A check doesn't report any Web Vitals. They are all `null`. This happens when for example when using Puppeteer based checks.
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:181]: typo: "when for example when"

**Review by ragog — CHANGES_REQUESTED** [review]
@tnolet this needs a bit of polish, mostly due to typos.
Also we missed "integratOns" on line 18 last time it seems.

File: site/content/docs/CICD/vercel.md (hunk 1 above)
Comment (tnolet) [reply site/content/docs/CICD/vercel.md:133]: I think this is fine. It makes the docs readable

**Comment by tnolet** [comment]
@ragog thanks for the great review. I addressed all concerns, except for the colloquialisms!

**Review by ragog — APPROVED** [review]
LGTM

**Pull request merged**